src/App.tsx

```diff
@@ -1,19 +1,81 @@
 import './App.css';
 import { GenreSelect } from './components/GenreSelect/GenreSelect';
 import { SearchForm } from './components/SearchForm/SearchForm';
-import { Counter } from './components/counter/Counter';
-
+import { Counter } from './components/Counter/Counter';
+import { MovieTile } from './components/MovieTile/MovieTile';
+import styled from 'styled-components';
+import { Colors } from './Colors';
+import { MovieDetails } from './components/MovieDetails/MovieDetails';
+import { MovieDetailed } from './types/movie-detailed';
+import { SortControl } from './components/SortControl/SortControl';
 const GENRE_LIST = ['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME'];
 
+const movie = {
```

Comment:
1) To reduce redundant constants - lets create a new folder `mocks` and then store there these constants.
2) Then use mocks for instance here and in tests (like `MovieDetails.test.tsx` `MovieTile.test.tsx`) and other places where/if we need them.